dev-util/scons/scons-3.0.5-r1.ebuild

```diff
@@ -42,14 +42,15 @@ src_unpack() {
 	# inside src/ subdirectory to make our life easier
 	if use test; then
 		unpack "${P}.gh.tar.gz"
-		rm -r "${P}/src" || die
+		find "${P}/src" -type f -name '*.py' -delete && echo "Done unpack"
```

Comment:
`||die`. This `echo` has no value.

Why do we need to remove them, by the way?

Reply:
Do you mean I need to replace `&& echo` by `|| die` here?

Initially the entire `"${P}src"` was removed in the early ebuilds (see e87d272).
And I tried to reproduce this but to fix Docbook tests.

I don't yet understand why but if I remove `find ... "*.py" ... -delete` command from my ebuild
then additional 45 tests are running from `/src/engine/Scons` directory and sub-directories.
 that I did't test early (totally 1200 tests instead of 1155). So I'm confused now.

The additional tests:
```
>>> Test phase: dev-util/scons-3.0.5-r2
 [32;01m*[0m python2_7: running distutils-r1_run_phase python_test
1/1200 (0.08%) /usr/bin/python2.7 -tt src/engine/SCons/ActionTests.py
..................sh: /usr/bin/python2.7_no_such_command_: No such file or directory
sh: /usr/bin: Is a directory
sh: /var/tmp/portage/dev-util/scons-3.0.5-r2/temp/testcmd.73.ceqPEk/outfile: Permission denied
....................................
----------------------------------------------------------------------
Ran 54 tests in 0.869s
OK
2/1200 (0.17%) /usr/bin/python2.7 -tt src/engine/SCons/BuilderTests.py
......................................
----------------------------------------------------------------------
Ran 38 tests in 0.379s
OK
3/1200 (0.25%) /usr/bin/python2.7 -tt src/engine/SCons/CacheDirTests.py
......
----------------------------------------------------------------------
Ran 6 tests in 0.130s
OK
4/1200 (0.33%) /usr/bin/python2.7 -tt src/engine/SCons/DefaultsTests.py
...
----------------------------------------------------------------------
Ran 3 tests in 0.005s
OK
5/1200 (0.42%) /usr/bin/python2.7 -tt src/engine/SCons/EnvironmentTests.py
.............................................................................................................................
----------------------------------------------------------------------
Ran 125 tests in 4.930s
OK
testFunc(["foo.out"], ["foo1.in", "foo2.in"])
6/1200 (0.50%) /usr/bin/python2.7 -tt src/engine/SCons/ErrorsTests.py
......
----------------------------------------------------------------------
Ran 6 tests in 0.001s
OK
7/1200 (0.58%) /usr/bin/python2.7 -tt src/engine/SCons/ExecutorTests.py
....................
----------------------------------------------------------------------
Ran 20 tests in 0.113s
OK
8/1200 (0.67%) /usr/bin/python2.7 -tt src/engine/SCons/JobTests.py
.......
----------------------------------------------------------------------
Ran 7 tests in 2.577s
OK
9/1200 (0.75%) /usr/bin/python2.7 -tt src/engine/SCons/MemoizeTests.py
..
----------------------------------------------------------------------
Ran 2 tests in 0.000s
OK
10/1200 (0.83%) /usr/bin/python2.7 -tt src/engine/SCons/Node/AliasTests.py
.......
----------------------------------------------------------------------
Ran 7 tests in 0.001s
OK
11/1200 (0.92%) /usr/bin/python2.7 -tt src/engine/SCons/Node/FSTests.py
.................................................................................
----------------------------------------------------------------------
Ran 81 tests in 2.566s
OK
Got error :[Errno 17] File exists: '/var/tmp/portage/dev-util/scons-3.0.5-r2/temp/testcmd.495.gqOLZx/work'
12/1200 (1.00%) /usr/bin/python2.7 -tt src/engine/SCons/Node/NodeTests.py
................................................................
----------------------------------------------------------------------
Ran 64 tests in 0.105s
OK
13/1200 (1.08%) /usr/bin/python2.7 -tt src/engine/SCons/Node/PythonTests.py
.......
----------------------------------------------------------------------
Ran 7 tests in 0.001s
OK
14/1200 (1.17%) /usr/bin/python2.7 -tt src/engine/SCons/PathListTests.py
..........
----------------------------------------------------------------------
Ran 10 tests in 1.056s
OK
15/1200 (1.25%) /usr/bin/python2.7 -tt src/engine/SCons/Platform/PlatformTests.py
....
----------------------------------------------------------------------
Ran 4 tests in 0.011s
OK
16/1200 (1.33%) /usr/bin/python2.7 -tt src/engine/SCons/Platform/virtualenvTests.py
........
----------------------------------------------------------------------
Ran 8 tests in 0.001s
OK
17/1200 (1.42%) /usr/bin/python2.7 -tt src/engine/SCons/SConfTests.py
.....................
----------------------------------------------------------------------
Ran 21 tests in 20.256s
OK
18/1200 (1.50%) /usr/bin/python2.7 -tt src/engine/SCons/SConsignTests.py
......
----------------------------------------------------------------------
Ran 6 tests in 0.005s
OK
19/1200 (1.58%) /usr/bin/python2.7 -tt src/engine/SCons/Scanner/CTests.py
..............
----------------------------------------------------------------------
Ran 14 tests in 0.274s
OK
20/1200 (1.67%) /usr/bin/python2.7 -tt src/engine/SCons/Scanner/DTests.py
...........
----------------------------------------------------------------------
Ran 11 tests in 0.080s
OK
21/1200 (1.75%) /usr/bin/python2.7 -tt src/engine/SCons/Scanner/DirTests.py
..
----------------------------------------------------------------------
Ran 2 tests in 0.075s
OK
22/1200 (1.83%) /usr/bin/python2.7 -tt src/engine/SCons/Scanner/FortranTests.py
................
----------------------------------------------------------------------
Ran 16 tests in 0.296s
OK
23/1200 (1.92%) /usr/bin/python2.7 -tt src/engine/SCons/Scanner/IDLTests.py
............
----------------------------------------------------------------------
Ran 12 tests in 0.278s
OK
24/1200 (2.00%) /usr/bin/python2.7 -tt src/engine/SCons/Scanner/LaTeXTests.py
...
----------------------------------------------------------------------
Ran 3 tests in 0.073s
OK
25/1200 (2.08%) /usr/bin/python2.7 -tt src/engine/SCons/Scanner/ProgTests.py
.........
----------------------------------------------------------------------
Ran 9 tests in 0.083s
OK
26/1200 (2.17%) /usr/bin/python2.7 -tt src/engine/SCons/Scanner/RCTests.py
...
----------------------------------------------------------------------
Ran 3 tests in 0.072s
OK
27/1200 (2.25%) /usr/bin/python2.7 -tt src/engine/SCons/Scanner/ScannerTests.py
..........................
----------------------------------------------------------------------
Ran 26 tests in 0.003s
OK
28/1200 (2.33%) /usr/bin/python2.7 -tt src/engine/SCons/Script/MainTests.py
----------------------------------------------------------------------
Ran 0 tests in 0.000s
OK
29/1200 (2.42%) /usr/bin/python2.7 -tt src/engine/SCons/Script/SConscriptTests.py
30/1200 (2.50%) /usr/bin/python2.7 -tt src/engine/SCons/SubstTests.py
...........................
----------------------------------------------------------------------
Ran 27 tests in 0.035s
OK
31/1200 (2.58%) /usr/bin/python2.7 -tt src/engine/SCons/TaskmasterTests.py
...............
----------------------------------------------------------------------
Ran 15 tests in 0.011s
OK
32/1200 (2.67%) /usr/bin/python2.7 -tt src/engine/SCons/Tool/JavaCommonTests.py
................
----------------------------------------------------------------------
Ran 16 tests in 0.005s
OK
33/1200 (2.75%) /usr/bin/python2.7 -tt src/engine/SCons/Tool/ToolTests.py
.
----------------------------------------------------------------------
Ran 1 test in 0.052s
OK
34/1200 (2.83%) /usr/bin/python2.7 -tt src/engine/SCons/Tool/javacTests.py
.........
----------------------------------------------------------------------
Ran 9 tests in 0.001s
OK
35/1200 (2.92%) /usr/bin/python2.7 -tt src/engine/SCons/Tool/msvsTests.py
NO RESULT for msvsTests.py:  'linux2' is not win32
36/1200 (3.00%) /usr/bin/python2.7 -tt src/engine/SCons/Tool/wixTests.py
.
----------------------------------------------------------------------
Ran 1 test in 0.005s
OK
37/1200 (3.08%) /usr/bin/python2.7 -tt src/engine/SCons/UtilTests.py
...........................................
----------------------------------------------------------------------
Ran 43 tests in 0.008s
OK
38/1200 (3.17%) /usr/bin/python2.7 -tt src/engine/SCons/Variables/BoolVariableTests.py
...
----------------------------------------------------------------------
Ran 3 tests in 0.000s
OK
39/1200 (3.25%) /usr/bin/python2.7 -tt src/engine/SCons/Variables/EnumVariableTests.py
...
----------------------------------------------------------------------
Ran 3 tests in 0.001s
OK
40/1200 (3.33%) /usr/bin/python2.7 -tt src/engine/SCons/Variables/ListVariableTests.py
...
----------------------------------------------------------------------
Ran 3 tests in 0.001s
OK
41/1200 (3.42%) /usr/bin/python2.7 -tt src/engine/SCons/Variables/PackageVariableTests.py
...
----------------------------------------------------------------------
Ran 3 tests in 0.002s
OK
42/1200 (3.50%) /usr/bin/python2.7 -tt src/engine/SCons/Variables/PathVariableTests.py
.......
----------------------------------------------------------------------
Ran 7 tests in 0.005s
OK
43/1200 (3.58%) /usr/bin/python2.7 -tt src/engine/SCons/Variables/VariablesTests.py
.............
----------------------------------------------------------------------
Ran 13 tests in 0.015s
OK
44/1200 (3.67%) /usr/bin/python2.7 -tt src/engine/SCons/WarningsTests.py
...
----------------------------------------------------------------------
Ran 3 tests in 0.001s
OK
45/1200 (3.75%) /usr/bin/python2.7 -tt src/engine/SCons/cppTests.py
..............................
----------------------------------------------------------------------
Ran 30 tests in 0.011s
OK
```